zowe/zowe-cli
# Fix secrets SDK package to include the right files

**What It Does**
The publish workflow succeeded for the Secrets SDK, but resulted in an 80MB TGZ containing the wrong files:
https://zowe.jfrog.io/ui/repos/tree/General/npm-local-release/@zowe/secrets-for-zowe-sdk/-/@zowe/secrets-for-zowe-sdk-7.18.0-next.1.tgz

* Restore `.npmignore` from @traeok's repo to exclude Rust binaries in `target` dir
* Move JS source into `lib` folder and include it in the package
* Include `Cargo.lock` for keyring package in this repo
* Consolidate multiple `.gitignore` files into one

**How to Test**
Check out this branch
Run `npm install` and `npm run build`
Then cd into `packages/secrets` and run `npm pack`
Verify that the list of files added to the TGZ looks good

**Review Checklist**
I certify that I have:

**Additional Comments**
